config/github.yml

```diff
@@ -1,60 +1,41 @@
-# listener services that you can opt into for a repository
 services:
     _defaults:
         autowire: true      # Automatically injects dependencies in your services.
+        autoconfigure: false
 
-    app.subscriber.status_change_by_comment_subscriber:
-        class: App\Subscriber\StatusChangeByCommentSubscriber
-        public: true
-
-    app.subscriber.status_change_on_push_subscriber:
-        class: App\Subscriber\StatusChangeOnPushSubscriber
-        public: true
-
-    app.subscriber.needs_review_new_pr_subscriber:
-        class: App\Subscriber\NeedsReviewNewPRSubscriber
-        public: true
-
-    app.subscriber.bug_label_new_issue_subscriber:
-        class: App\Subscriber\BugLabelNewIssueSubscriber
-        public: true
-
-    app.subscriber.auto_label_pr_from_content_subscriber:
-        class: App\Subscriber\AutoLabelPRFromContentSubscriber
-        public: true
-
-    app.subscriber.status_change_by_review_subscriber:
-        class: App\Subscriber\StatusChangeByReviewSubscriber
-        public: true
+    App\Subscriber\:
+        resource: '../src/Subscriber/*'
 
 parameters:
+    env(SYMFONY_DOCS_SECRET): ''
+    env(SYMFONY_SECRET): ''
```

Comment:
Since we have these both defined in `.env`, I don't think we also need to give them defaults here